sigstore-java/src/main/java/dev/sigstore/bundle/BundleFactory.java

```diff
@@ -51,4 +54,19 @@ public static String createBundle(KeylessSigningResult signingResult) {
           "Can't serialize signing result to Sigstore Bundle JSON", e);
     }
   }
+
+  /**
+   * Read a bundle json and convert it back into a keyless signing result for use within this
+   * library
+   *
+   * @param jsonReader a reader to a valid bundle json file
+   * @return the converted signing result object
+   */
+  public static KeylessSigningResult readBundle(Reader jsonReader) {
+    try {
+      return BundleFactoryInternal.readBundle(jsonReader);
+    } catch (IOException | CertificateException e) {
```

Comment:
should the method just throw IOException?

Reply:
good point, added BundleParseException